### PR TITLE
Added Romanian translation [ro_RO]

### PR DESCRIPTION
**Pull request by florinsdistortedvision**
Full Romanian language with Romanian diacritics including index.html in ro folder, CODEOWNERS and README update.

### CODE REVIEW

**Comment by nitanmarcel** [comment]
There's an untranslated word "actually" in `Ce vrea această persoană actually să întrebe este,`

Everything else looks ok.

**Comment by florinsdistortedvision** [comment]
Fixed the actually word not being translated!

**Comment by nitanmarcel** [comment]
> Fixed the actually word not being translated!

Also there's a phrase. `[...] cu ce problemă am`. Wouldn't be better `cu problema mea`?

**Comment by florinsdistortedvision** [comment]
I guess both can work, but I did translate it as close the original as possible.

**Comment by nitanmarcel** [comment]
> I guess both can work, but I did translate it as close as the original as possible.

Ah okay. We'll keep it like this then.

@tulir this is safe to merge now :)

**Review by tulir — APPROVED** [review]

File: ro/index.html [hunk 1]
```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (tulir) [inline ro/index.html:2]:
```suggestion
<html lang="ro">
```

**Pull request merged**